docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[data-views-in-sec]]
+= {Data-sources} in Elastic Security
+
+The active {data-source} determines which data appears throughout {es-sec}.
+{Data-sources} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+
+== How to switch {data-sources}
+
+You can tell which {data-source} is active by clicking the {Data-source} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, or Hosts.
+To switch to another existing {data-source}, click the same button {Data-source} and choose a different {data-source} under `Choose {data-source}`:
+
+image::images/dataview-default-setting-highlighted.png[the securitySolution:defaultIndex setting, shown in context]
+
+
+== How to create or modify {data-sources}:
+
+To modify or delete an existing {data-source} or to define a new one, you need the required permissions.
+To learn more, refer to {apm-app-ref}/data-views.html[{kib} {Data-Sources}].
+
+
+== The default {data-source}
+
+The first time you add data to an {es-sec} instance within a given {apm-app-ref}/xpack-spaces.html[space], the default {data-source} gets generated for that space and becomes active.
+If you delete the active {data-source} when there are no other defined {data-sources}, the default {data-source} will regenerate and become active upon the refresh of any {es-sec} page in that space, because the app requires a {data-source} to function.
+
+The default {data-source} is defined by the `securitySolution:defaultIndex` setting, which you can modify from the Security Solution section of {kib}'s advanced settings (**Stack Management** > **Advanced Settings** > **Security Solution**):
+
+image::images/dataview-button-highlighted.png[detail view of the Data view button]
+
+The preset value for `securitySolution:defaultIndex` is (comma-delimited): `apm-*-transaction*,auditbeat-*,endgame-*,filebeat-*,logs-*,packetbeat-*,winlogbeat-*`.
+This includes, for example, all data shipped via {beats} or the {agent}.
```

Comment:
You can probably do away with this information and link user to https://www.elastic.co/guide/en/security/master/advanced-settings.html#update-sec-indices instead. This section might also need to be refreshed a bit to mention how data views and index patterns intersect.

Also, if you do link to the "Advance settings" section, you can probably remove the `dataview-default-setting-highlighted.png` image since that topic already has a screenshot to show where the advanced Security settings are located.

Reply:
I think you're right, and that would be easier to maintain. I'm definitely open to it, but I'm torn about whether that would be too much linking. I feel like the page is a reasonable enough length that it might be okay to keep it in.

Reply:
I'm in favor of linking to the existing content in Advanced Settings, for maintainability and to keep this new topic focused on the general idea of data views in the Security app.